src/enclave/App/SGXEnclave.h

```diff
@@ -53,15 +53,9 @@ extern "C" {
   Java_edu_berkeley_cs_rise_opaque_execution_SGXEnclave_NonObliviousAggregateStep2(
     JNIEnv *, jobject, jlong, jbyteArray, jbyteArray, jbyteArray, jbyteArray, jbyteArray);
 
-  JNIEXPORT jbyteArray JNICALL Java_edu_berkeley_cs_rise_opaque_execution_SGXEnclave_RemoteAttestation0(
-    JNIEnv *, jobject, jlong);
-
   JNIEXPORT jbyteArray JNICALL Java_edu_berkeley_cs_rise_opaque_execution_SGXEnclave_RemoteAttestation1(
```

Comment:
Do we still need RemoteAttestation1 and RemoteAttestation3? If so, maybe we could rename them to avoid a confusing gap in numbers.

Reply:
I've renamed these functions.